lib/pbench/test/unit/server/database/test_api_key.py

```diff
@@ -0,0 +1,64 @@
+import datetime
+
+import pytest
+
+from pbench.server.database.database import Database
+from pbench.server.database.models.api_key import APIKey
+from pbench.test.unit.server.database import FakeSession
+
+
+class TestAPIKey:
+
+    session = None
+
+    # Current time to encode in the payload
+    current_utc = datetime.datetime.now(datetime.timezone.utc)
+    expiration = current_utc + datetime.timedelta(minutes=10)
+
+    @pytest.fixture()
+    def fake_db(self, monkeypatch, server_config):
+        """
+        Fixture to mock a DB session for testing.
+
+        We patch the SQLAlchemy db_session to our fake session. We also store a
+        server configuration object directly on the Database.Base (normally
+        done during DB initialization) because that can't be monkeypatched.
+        """
+        __class__.session = FakeSession(APIKey)
+        monkeypatch.setattr(Database, "db_session", __class__.session)
+        Database.Base.config = server_config
+        yield monkeypatch
+
+    def test_construct(self, fake_db, create_drb_user):
+        """Test api_key constructor"""
```

Comment:
Why do we use `fake_db` here instead of `db_session`, like we did in the subsequent tests?